apps/prairielearn/src/lib/course.ts

```diff
@@ -38,7 +37,6 @@ export async function checkBelongsAsync(
     throw new Error('access denied');
```

Comment:
Can we throw an `HttpStatusError` with 403 here?

Reply:
Done in 8184cac3ab76b6f0060f9c0f2e622e534fb03ab8.